pkg/operator/mon/mon_test.go

```
@@ -51,11 +51,21 @@ func TestStartMonPods(t *testing.T) {
 		},
 	}
 	context := &clusterd.Context{
-		KubeContext: kit.KubeContext{Clientset: clientset, MaxRetries: 1},
-		Executor:    executor,
-		ConfigDir:   configDir,
+		Clientset: clientset,
+		Executor:  executor,
+		ConfigDir: configDir,
+	}
+	c := &Cluster{
+		context:             context,
+		dataDirHostPath:     "",
+		Namespace:           namespace,
+		Version:             "myversion",
+		Size:                3,
+		maxMonID:            -1,
+		waitForStart:        true,
+		monPodRetryInterval: 1 * time.Second,
```

Sure.. is this to fail fast?